<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TMS</class>
 <widget class="QMainWindow" name="TMS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>835</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
font: 10pt &quot;푸라닭 젠틀 고딕&quot;;

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="localBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <item>
     <property name="text">
      <string>1160파주</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1120청주</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1110중앙</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1130구미</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1140광주</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1142여수</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1150창원</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1141익산</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1190미국</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1151울산</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>281</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 36pt &quot;강원교육튼튼&quot;;</string>
    </property>
    <property name="text">
     <string>TMS - GA</string>
    </property>
   </widget>
   <widget class="QLabel" name="local">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;푸라닭 젠틀 고딕&quot;;</string>
    </property>
    <property name="text">
     <string>지역</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="no_over_cap">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>210</y>
      <width>96</width>
      <height>19</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>과적 금지</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn1">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>680</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>실행</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="no_over_time">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>141</width>
      <height>19</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>초과 근무 금지</string>
    </property>
   </widget>
   <widget class="QLabel" name="quantity">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>284</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>물량 비용</string>
    </property>
   </widget>
   <widget class="QLabel" name="hub_radius">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>314</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>반경 비용</string>
    </property>
   </widget>
   <widget class="QLabel" name="crossline">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>344</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>교차 비용</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="max_capacity_cost">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>380</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="value">
     <double>0.800000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="max_capacity">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>380</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>최대 적재</string>
    </property>
   </widget>
   <widget class="QLabel" name="limit_capacity">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>420</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>제한 용량</string>
    </property>
   </widget>
   <widget class="QLabel" name="limit_time">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>450</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>최대 시간</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="limit_capacity_var">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>420</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>5</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="limit_time_var">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>450</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>5</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="max_capacity_cost_var">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>170</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>최대 중량 비용</string>
    </property>
   </widget>
   <widget class="QLabel" name="timewindow">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>210</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>시간 초과 비용</string>
    </property>
   </widget>
   <widget class="QLabel" name="car_speed">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>250</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>차량 속도(km/h)</string>
    </property>
   </widget>
   <widget class="QLabel" name="loading_unloading">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>290</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>상하차 시간(m)</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="loading_unloading_var">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>290</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="used_limit_time">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>330</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>사용된 근무 시간의</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="used_limit_time_var">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>330</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="value">
     <double>0.700000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="used_limit_time2">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>330</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>% 를 충족하면 차량을 제외</string>
    </property>
   </widget>
   <widget class="QLabel" name="used_limit_cap2">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>370</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>% 를 충족하면 차량을 제외</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="used_limit_cap_var">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>360</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="value">
     <double>0.700000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="used_limit_cap">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>370</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>사용된 차 용량의</string>
    </property>
   </widget>
   <widget class="QLabel" name="car_customer">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>500</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>차와 지점간의 거리 비용</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="quantity_var">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>280</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="hub_radius_var">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>310</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="crossline_var">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>340</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="car_customer_var">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>500</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="singleStep">
     <number>10000</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="max_capacity_cost_var_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>170</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="timewindow_var">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>200</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="car_speed_var">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>250</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>150</number>
    </property>
    <property name="value">
     <number>70</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="available_yong">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>210</y>
      <width>96</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>용차 사용</string>
    </property>
   </widget>
   <widget class="QLabel" name="count">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>410</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>초기 유전 생성 반복 횟수</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="count_var">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>410</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLabel" name="yong_car_cost">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>450</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>용차 비용</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="yong_car_var">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>450</y>
      <width>43</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="mutant">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>490</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>돌연변이 확률</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="mutant_var">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>490</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="value">
     <double>0.300000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="change_var">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>530</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="value">
     <double>0.500000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="change">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>530</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>교차 확률</string>
    </property>
   </widget>
   <widget class="QLabel" name="num_of_generation">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>540</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>반복 세대 수 제한</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="num_of_generation_var">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>540</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="no_update_var">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>580</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="no_update">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>580</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>까지 업데이트가 되지 않으면 종료</string>
    </property>
   </widget>
   <widget class="QLabel" name="yong_car_select">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>570</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>용차 선택 확률</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="yong_car_select_var">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>570</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>951</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
